torch/cuda/memory.py

```diff
@@ -72,6 +72,33 @@ def caching_allocator_delete(mem_ptr):
     torch._C._cuda_cudaCachingAllocator_raw_delete(mem_ptr)
 
 
+def set_memory_fraction(fraction, device: Union[Device, int] = None) -> None:
+    r"""Set memory fraction for a device.
+    The fraction is used to limit allocated memory on a CUDA device.
```

Comment:
The statement is incorrect as you are only limiting memory used for caching allocator, so running tow processes with fraction of 0.5 not going to be possible (but expected as per description) as some of the memory going to be used by operators and cuda context.

Reply:
@VitalyFedyunin This does not work for multiprocess limiting, becasue all subproesses will get its own space  and do not share info to each other. However, in most cases, one can use it to split memory, if he/she knows put the setting in subprocess.  So should I just change the statement or do some more, such as to deal with the multiprocess situation?

Reply:
@VitalyFedyunin hi, I've just changed the statment and the name of this function, to make it more clearly.